Export templates dont compile Godot 3.0, Godot 2.x its fine

Build error in Godot 3.0, Godot 2.x its fine

Antergos 64Bits
```
[100%] Linking Shared Library ==> bin/libgodot.android.opt.armv7.neon.so
[100%] Move("platform/android/java/libs/release/armeabi-v7a/libgodot_android.so", "bin/libgodot.android.opt.armv7.neon.so")
[100%] scons: done building targets.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'java'.
> Could not resolve all dependencies for configuration ':_debugApkCopy'.
   > Could not find any version that matches com.android.support:support-v4:27.+.
     Versions that do not match:
         26.0.0-alpha1
         25.3.1
         25.3.0
         25.2.0
         25.1.1
         + 32 more
     Searched in the following locations:
         file:/mypath/Android/Sdk/extras/android/m2repository/com/android/support/support-v4/maven-metadata.xml
         file:/otherpath/godot-master/platform/android/java/sdk-manager/com/android/support/support-v4/maven-metadata.xml
         file:/otherpath/godot-master/platform/android/java/sdk-manager/com/android/support/support-v4/
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
[machine@user java]$ 
```

--- a/platform/android/build.gradle.template
+++ b/platform/android/build.gradle.template
@@ -15,6 +15,7 @@ allprojects {
     repositories {
     	jcenter()
 	mavenCentral()
+	google()
 	$$GRADLE_REPOSITORY_URLS$$
     }
 }

